pkgs/development/libraries/libipfix/default.nix

```diff
@@ -0,0 +1,16 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation {
+  name = "libipfix";
+  version = "110209";
+  src = fetchurl {
+    url = " http://sourceforge.net/projects/libipfix/files/libipfix/libipfix_110209.tgz";
+    sha256 = "0h7v0sxjjdc41hl5vq2x0yhyn04bczl11bqm97825mivrvfymhn6";
+  };
+  meta = {
+    homepage = http://libipfix.sourceforge.net/;
+    description = "The libipfix C-library implements the IPFIX protocol defined by the IP Flow Information Export working group of the IETF.";
+    license = stdenv.lib.licenses.lgpl3;
+    platforms = stdenv.lib.platforms.linux;
```

Comment:
Do you want to maintain this package?

Reply:
yes, added myself. 
I've the feeling it will be easy to maintain it: no more commit since 6years :)